examples/data-visualization/index.ts

```diff
@@ -0,0 +1,165 @@
+import { Viewer, NavigationCube, Heatmap, ContinuousHeatmapChannel, HeatmapSource, Icons, CameraType, ViewType, IHeatmapChannel, ClippingPlane, ProductType, } from '../..';
+import { Icon } from '../../src/plugins/DataVisualization/Icons/icon';
+import { Grid } from '../../src/plugins/Grid/grid';
+import { IconsData } from './icons';
+
+const viewer = new Viewer("viewer");
+const cube = new NavigationCube();
```

Comment:
For simplicity, lets remove the Grid and Cube from the example. Covered elsewhere